### ISSUE TITLE
Issue with using `boolean()` from refine pckg

### ISSUE THREAD

**Issue by enisze** [facebookexperimental/Recoil#1922]
```
TypeError: (0 , _recoiljs_refine__WEBPACK_IMPORTED_MODULE_0__.boolean) is not a function

This error happened while generating the page. Any console logs will be displayed in the terminal window.
```

This happens in the `urlSyncEffect` / `syncEffect` in an atom `effects` prop.
Using `number()` or `string()` does not cause any problem. Just boolean. Any ideas?

**Comment by zoeyfyi** [comment]
I get a similar error

```
Failed to compile.

Attempted import error: 'boolean' is not exported from '@recoiljs/refine' (imported as 'refine').
```

Code to reproduce: https://runkit.com/embed/3o2k1gs2vx3i

**Comment by osiux** [comment]
Seems like something on the build process strips this export. The result is that every build (cjs, es, umd) has no export of the `boolean` function. Unfortunately haven't been able to find why this happens.

**Comment by drarmstr** [comment]
Interesting.  This issue doesn't show up with our internal builds so it looks like something with the open source build.  `boolean()` is exported the same as `number()` and `string()`, but `boolean` is actually a reserved word in JS.  So, that may likely be the issue.  I think we'll have to find an alternative name for it.  (cc @bsouthga)

**Comment by osiux** [comment]
@nichita-pasecinic the new exported names is `bool`, since `boolean` is a reserved word.

**Comment by nichita-pasecinic** [comment]
@osiux I got it, but when the new release will be published (`0.1.1.`) ?

**Comment by drarmstr** [comment]
Hopefully today.
